bin/middlewares/og_meta.js

```diff
@@ -1,94 +1,92 @@
-const request = require('request')
+const request = require('request');
 
 
 
```

Comment:
Why 3 blank lines btw?

Reply:
I just added the ";", the blank lines were already there... but fixed!